### ISSUE TITLE
Logitech Revue (Google TV)

### ISSUE THREAD

**Issue by zarthan** [MythTV-Clients/MythTV-Android-Frontend#22]
The App installs and launches. Only an away profile shows on the initial screen. My Samsung SII displays a home and an away without configuring anything.

Adding a profile the backend address is displayed in the backend address text box (http://192.168.111.15:6544/ ) I enter a name for the profile and click save. I get an "invalid MythTV Master Backend Address" popup. A scan network gives me a force close popup with "The application MythTV (process org.mythtv) has stopped unexpectedly. Please try again."  

I would love to get this working on GoogleTV.

**Comment by dmfrey** [comment]
@zarthan It is my plan to get it on google tv as well.  I have 2 of them and would love for this to work.  I will address google tv in due time.  I believe I need to add some extra libraries to support google tv.

Btw, how are you installing it on a google tv?  just putting the apk on a usb stick? downloading it from the site right on the google tv?  Do you have to enable any android development modes on google tv like you have to do on a phone or tablet?

**Comment by zarthan** [comment]
On Wed, Jun 13, 2012 at 8:35 AM, Daniel Frey <
reply@reply.github.com

> wrote:
> 
> @zarthan It is my plan to get it on google tv as well.  I have 2 of them
> and would love for this to work.  I will address google tv in due time.  I
> believe I need to add some extra libraries to support google tv.
> 
> Btw, how are you installing it on a google tv?  just putting the apk on a
> usb stick? downloading it from the site right on the google tv?  Do you
> have to enable any android development modes on google tv like you have to
> do on a phone or tablet?
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> 
> https://github.com/MythTV-Android/mythtv-for-android/issues/22#issuecomment-6301042

The first time I installed the app I added it to Dropbox from my PC and
then opened it from the Dropbox app on GTV. The second time I downloaded it
using Chrome and installed directly. I needed to uninstall the first one
before I could install the one from Git because GTV detected a signature
change. I have install from Unknown Sources enabled since I have the Amazon
App Store installed. Looking forward. Let me know how I can help. I'm not a
programmer but I can sometimes figure things out.

**Comment by bthule** [comment]
The Revues cannot play MPEG-2.  That's not an issue if you transcode your recordings, but I personally don't like transcoding, it's time consuming and wastes energy (unless you are going to watch the transcoded video a bunch of times).

The Sony Google TV units do play MPEG-2, and I bought one and using the Avia media player, the picture quality is very good(-- but the interface and controls are lackluster.  

I am interested in helping develop this app to help it work on sony google tv units, but I am not sure the status, and if work is already going on to this end.  Is there a dev mailing list?

**Comment by drhill** [comment]
Semi-bump to show interest, but I'm also curious about live tv support.  I just downloaded the app on the market for my GNex and Xoom and was able to get recordings working with Dice Player.  But I bought my HDHomerun Prime and Google TV box (Sony blu ray) with the intention of replacing the old ugly cable box one day.  Can't do that without live tv as 90% of what I watch is sports. 

Personally I've been spent a decent bit of time writing an ugly app to get my Sony box to work with my Prime by hdhomerunlib (ported to java) and then dlna, but have been met with less then great results.  I noticed months ago the mythtv .25 api's had live streaming support, but it only mentioned transcoded streams (yuck!).  With the google tv devices that support the streams in hardware (mpeg2 for the nsz-gt1) it unnecessary to transcode the streams and would bog down a server.
